index.html

```diff
@@ -36,6 +36,40 @@ <h2>Donating To Us</h2>
 		<div class="content-divider">
 		</div> <!-- End of content-divider -->
 
+		<div class="card-component-container">
+			<h2>More Ways To Help</h2>
```

Comment:
This will look better centred (through css).

Reply:
agreed and done